lib/project_types/script/commands/enable.rb

```diff
@@ -59,6 +61,8 @@ def acquire_configuration(config_file: nil, config_props: nil)
           })
         end
         configuration
+      rescue Errno::ENOENT, Psych::SyntaxError
+        raise Errors::InvalidConfigYAMLError.new, ''
```

Comment:
We currently have a pattern of adding in accessors to an error instance if we need to read certain values in error messages. Look at this [error](https://github.com/Shopify/shopify-app-cli/blob/b671d1aee49656130689f9f5b2f86fde15e48c2d/lib/project_types/script/errors.rb#L9) and this [error message](https://github.com/Shopify/shopify-app-cli/blob/b671d1aee49656130689f9f5b2f86fde15e48c2d/lib/project_types/script/ui/error_handler.rb#L71) for example. Could we add a `config_file` attribute to this error, and avoid the interface changes to `error_handler.rb`?